### DISCUSSION TITLE
Is it possible to avoid the "Junk Screen" when powering the ESP?

### DISCUSSION THREAD

**Discussion by Murloc992** [Bodmer/TFT_eSPI#961] [Q&A - General]
As the title says, I am wondering if it is possible to avoid the junk screen (many random pixels, white and colored) that appears for a brief period of time?

The driver used is `ST7789_DRIVER`. Maybe there is something that I can feed before the library is initialized? Or maybe it's just inherent issue of the display/driver?

The screen I am talking about:

![image](https://user-images.githubusercontent.com/3779847/105992516-e73bd980-60ad-11eb-96c1-d6aee6a2f1f9.png)

Subsequent soft restarts do not have this junk screen, only during power on.

**Comment by Murloc992** [comment]
I have actually temporarily solved it by playing with the backlight pin. Disabling it for ~100ms during turn on hides the issue pretty well. 😄

**Comment by Bodmer** [comment]
Unfortunately it takes time to configure the display after power up. Not all displays show garbage at power up, most show a black or white screen.
